plugin-server/src/cdp/legacy-plugins/_destinations/posthog-avo/template.ts

```diff
@@ -0,0 +1,78 @@
+import { LegacyDestinationPlugin } from '../../types'
+import { onEvent, setupPlugin } from './index'
+
+// NOTE: This is a deprecated plugin and should never be shown to new users
+
+export const avoPlugin: LegacyDestinationPlugin = {
+    setupPlugin: setupPlugin as any,
+    onEvent,
+    template: {
+        free: false,
+        status: 'deprecated',
+        type: 'destination',
+        id: 'plugin-posthog-avo-plugin',
+        name: 'Avo Inspector Plugin',
+        description: 'Export PostHog events to Avo inspector.',
+        icon_url: 'https://raw.githubusercontent.com/PostHog/posthog-avo-plugin/main/logo.png',
+        category: [],
+        hog: 'return event',
+        inputs_schema: [
+            {
+                key: 'avoApiKey',
+                description: '',
+                type: 'string',
+                default: '',
+                required: true,
+                secret: true,
+            },
+            {
+                key: 'environment',
+                description: '',
+                type: 'string',
+                default: 'dev',
+                required: true,
+                secret: true,
```

Comment:
These aren't secret in the plugin.json?

Seems like there are a lot of copy paste errors making it hard for me to trust this PR. If you could do a really close check that would give me more confidence please